config/metric_config.go

```diff
@@ -60,7 +59,7 @@ type PrometheusConfig struct {
 }
 
 type Exporter struct {
-	Enabled *bool `default:"false" yaml:"enabled" json:"enabled,omitempty" property:"enabled"`
+	Enabled *bool `default:"true" yaml:"enabled" json:"enabled,omitempty" property:"enabled"`
```

Comment:
this will cause `url.getParam(constant.PrometheusExporterEnabledKey)` allways true, see https://github.com/apache/dubbo-go/blob/main/config/metric_config.go#L157

Reply:
I think the `toURL()` method should not add `&&` or `||` to change enabled logic, instead, only performs a format conversion operation.

Reply:
`metrics.Init(mc.toURL())` executed only when mc.Enable is true , I have no problem here.

Reply:
`metrics.Init(mc.toURL())` will executed only when `mc.Enable` is true, so I have no problem here.

Reply:
Got it, thanks.